audit/format.go

```diff
@@ -367,6 +369,7 @@ type AuditAuth struct {
 	DisplayName string            `json:"display_name"`
 	Policies    []string          `json:"policies"`
 	Metadata    map[string]string `json:"metadata"`
+	NumUses     int               `json:"num_uses,omitempty"`
```

Comment:
Policies/display name/metadata won't change. I wonder if on the request side, because this value does change, it should be `RemainingUses`. This could be accomplished by having two fields, both with `omitempty`. The downside is any parsing logic will have to understand both fields; on the flip side, the fields will individually be more meaningful.

Thoughts?